mozci/data/sources/treeherder/__init__.py

```diff
@@ -54,7 +99,18 @@ def normalize(cls, jobs):
                     job.submit_time, job.start_time, job.end_time
                 ),
             }
-            items.append(result)
+
+            result_map = {
+                "success": "passed",
+                "testfailed": "failed",
+                "bustage": "failed",
+                "usercancel": "canceled",
+                "retry": "exception",
+            }
```

Comment:
treeherder too doesn't support superseded, maybe we can ignore superseded tasks altogether in the taskcluster data source too?

Reply:
Same as above, it does it just doesn't need translation.

Reply:
Might as well keep it I guess... maybe we might want to get data on how often superseding happens?